turbo.json

```
@@ -0,0 +1,39 @@
+{
+  "pipeline": {
+    "codegen:entrypoints": {
+      "outputs": [
+        "links",
+        "shared",
+        "ssg",
+        "adapters",
+        "dist",
+        "http",
+        "observable",
+        "rpc",
+        "subscription"
+      ]
+    },
+    "build": {
+      "dependsOn": ["codegen:entrypoints"],
```

```suggestion
      "dependsOn": ["^build", "codegen:entrypoints"],
```

I think build should be in here, not sure how it handles devDeps though